rest_framework/templatetags/rest_framework.py

```diff
@@ -245,6 +245,29 @@ def items(value):
 
 
 @register.filter
+def schema_links(section, sec_key=None):
+    """
+    Recursively find every link in a schema, even nested.
+    """
+    NESTED_FORMAT = '%s > %s'
```

Comment:
works fine in firefox and chrome under debian x64

e.g:
`http://localhost:8000/docs/#users-animals > dogs > list`

also clicking the links on the sidebar work.